cherab/solps/plotting.py

```diff
@@ -0,0 +1,100 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib.collections import PolyCollection
+
+
+def plot_quadrangle_mesh(mesh, solps_data=None, ax=None):
+    """
+    Plot the quadrangle mesh grid geometry to a matplotlib figure.
+
+    :param mesh: SOLPSMesh object
+    :param solps_data: Data array defined on the SOLPS mesh (optional)
+    :param ax: matplotlib axes (optional)
+    :return: matplotlib axes
+    """
+    if ax is None:
+        _, ax = plt.subplots(constrained_layout=True)
+
+    if solps_data is None:
+        collection_mesh = create_quadrangle_polycollection(mesh, facecolor="none", edgecolor='b', linewidth=0.5)
+    else:
+        collection_mesh = create_quadrangle_polycollection(mesh, solps_data)
+    ax.add_collection(collection_mesh)
+
+    ax = format_matplotlib_axes(ax, mesh)
+    return ax
+
+
+def create_quadrangle_polycollection(mesh, solps_data: np.ndarray = None, **collection_kw):
```

Comment:
Python typing is supported from the version 3.5, but the core package supports python 3 in general (if I'm not mistaken, @jacklovell will probably confirm). This would mean you have to remove typing for now.